CrossPare/src/main/java/de/ugoe/cs/cpdp/eval/AbstractWekaEvaluation.java

```diff
@@ -507,4 +538,79 @@ public static double getNECM(Evaluation eval, double factor) {
         return (eval.numFalsePositives(1)+factor*eval.numFalseNegatives(1))/
                 (eval.numInstances());
     }
+
+    /**
+     * <p>
+     * Calculates the bias of a data set. That means the percentage of instances in
+     * the training data that are defective.
+     * </p>
+     *
+     * @param data data set to calculate the bias for
+     * @return bias
+     */
+    public static double getBias(Instances data) {
+        double count = 0.0;
+        for (Instance instance : data) {
+            if (instance.classValue() > 0.0) {
+                count += 1.0;
+            }
+        }

```

Comment:
data(data.classIndex()).nominalCounts should already implement this

Reply:
I changed it to use the nominalCounts attribute. But not all 'Instances' are 'nominal' at that point.

Reply:
Ah, right. In that case your solution was better and I guess you should revert this. Sorry, I forgot about that case. The rest looks good now.